Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -460,85 +460,31 @@ def merge_json_files(json_files)
     end
   end
 
+  def create_bottle_spec(bottle_hash)
```

Comment:
Sorry, I'd still like to see no changes to `Library/Homebrew/dev-cmd/bottle.rb` before tests are added. It's still pretty hard to review what has changed/not here.